public class OOPS_17_Exceptionhandling {
    public static void main(String[] args) {
        int a = 5;
        int c = 0;
        try{
            System.out.println(a/c);
        }
        catch (Exception l ){
            c= 5;
            System.out.println("kastro  the output is creating error ");

        }
    }
}
